plugins/scroll-options/src/index.js

```diff
@@ -31,5 +31,61 @@ export class Plugin {
   /**
    * Initialize.
    */
-  init() { }
+  init() {
+    const dragSurface = this.workspace_.getBlockDragSurface();
+    Blockly.bindEventWithChecks_(
+        dragSurface.getSvgRoot(), 'wheel', this, this.onMouseWheel_);
+  }
+
+  onMouseWheel_(e) {
```

Comment:
This needs jsdoc (I know there are already 4 warnings about this but they kind of blend in with all the warnings we have to ignore)